### PR TITLE
 Play 2.7.0 released! Comes with Scala 2.13.0-M5 ;)

### PR DESCRIPTION
**Pull request by mkurz**
@mumoshu Can you also release `0.10.0`? That would be really great :+1: 

Don't forget to update the version in all the `build.sbt` files (removing the `-RC3`):
https://github.com/mumoshu/play2-memcached/blob/66d7412158da83b1c33b326110ca82ea412fc8bd/build.sbt#L2
https://github.com/mumoshu/play2-memcached/blob/66d7412158da83b1c33b326110ca82ea412fc8bd/samples/java/build.sbt#L5
https://github.com/mumoshu/play2-memcached/blob/66d7412158da83b1c33b326110ca82ea412fc8bd/samples/scala/build.sbt#L5

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,12 @@ val spymemcached    = "net.spy" % "spymemcached" % "2.12.3"
...
 lazy val baseSettings = Seq(
-  scalaVersion := "2.12.7",
-  crossScalaVersions := Seq("2.11.12", "2.12.7"),
-  parallelExecution in Test := false
+  scalaVersion := "2.12.8",
+  crossScalaVersions := Seq("2.11.12", "2.12.8", "2.13.0-M5"),
+  parallelExecution in Test := false,
+  // Workaround until omnidoc gets published for Scala 2.13
+  // http://central.maven.org/maven2/com/typesafe/play/play-omnidoc_2.13/
+  PlayKeys.playOmnidoc := false
```
Comment (halfninja) [inline build.sbt]: Could keep the regular behaviour of omnidoc for non-2.13 versions if anyone was bothered.

`PlayKeys.playOmnidoc := PlayKeys.playOmnidoc.value && !scalaBinaryVersion.value.startsWith("2.13")`

**Comment by halfninja** [comment]
Have tested `0.10.0-RC3` against some of our apps and Play 2.7.0 and it works fine so I think this is ready to roll.

File: build.sbt (hunk 1 above)
Comment (mkurz) [reply build.sbt]: Your example code doesn't work. I get errors. Please try yourself.

Comment (mkurz) [reply build.sbt]: Also I think it does not really matter for a module like play2-memcached if `playOmnidoc` is used or not. `playOmnidoc` does not make any use of it anyway and usually it only matters in the application itself (and even there usually no one makes us of it, despite maybe running the Play docs offline).

Comment (halfninja) [reply build.sbt]: You're quite right that it doesn't work here - I'd been testing it on a project with the Play plugin enabled, but it doesn't work here in `commonSettings` as there's no previous value of `playOmnidoc` to work from.

You're also right that it doesn't really matter for the reasons you describe. If it was an issue then you'd be able to use this in the projects that did use the plugin (I think it's just `javaSample` and `scalaSample` that are affected) but I think your PR is good to go as-is.

Comment (mkurz) [reply build.sbt]: :+1:

**Comment by mkurz** [comment]
Ping @mumoshu

**Comment by mkurz** [comment]
@mumoshu Can you please give a short reply if you have time to to merge this pull request and release a new version anytime soon? Thanks!

**Comment by mkurz** [comment]
@mumoshu I saw you released 0.10.0 - can you also git-tag this release and push the tag upstream? Thanks!

**Comment by mumoshu** [comment]
@mkubala (How did you notice that?!) Yes! Just finished the release and tagged v0.10.0. Thanks again for your support ☺️

**Pull request merged**